live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/utils/RepoEventTestUtils.java

```diff
@@ -0,0 +1,65 @@
+package org.alfresco.hxi_connector.live_ingester.messaging.in.utils;
+
+import static lombok.AccessLevel.PRIVATE;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import lombok.NoArgsConstructor;
+
+import org.alfresco.repo.event.v1.model.DataAttributes;
+import org.alfresco.repo.event.v1.model.EventType;
+import org.alfresco.repo.event.v1.model.NodeResource;
+import org.alfresco.repo.event.v1.model.RepoEvent;
+
+@SuppressWarnings({"PMD.TestClassWithoutTestCases"})
+@NoArgsConstructor(access = PRIVATE)
+public class RepoEventTestUtils
+{
+
+    public static void setTime(RepoEvent<DataAttributes<NodeResource>> event, long timestamp)
+    {
+        given(event.getTime()).willReturn(dateFromTimestamp(timestamp));
+    }
+
+    public static ZonedDateTime dateFromTimestamp(long timestamp)
+    {
+        return ZonedDateTime.ofInstant(Instant.ofEpochMilli(timestamp), ZoneId.of("UTC"));
+    }
+
+    public static void setType(RepoEvent<DataAttributes<NodeResource>> event, EventType type)
+    {
+        given(event.getType()).willReturn(type.getType());
+    }
+
+    public static void setNodeResource(RepoEvent<DataAttributes<NodeResource>> event, NodeResource nodeResource)
```

Comment:
This method (and `setNodeResourceBefore`) need a bit of knowledge to use correctly. I'd suggest either or both of:
1. Add some javadoc to explain what they do and how to use them (e.g. `event.getData()` must return a mock).
2. Move the methods back into the test class where they are used (i.e. don't claim they're general purpose).

Reply:
I am moving these methods to individual test classes then